tests/legacy/system/test_edb_config_json.py

```diff
@@ -60,6 +60,48 @@ def test_01_create_edb(self, edb_examples):
 
         edbapp.close()
 
+    def test_01a_setups_frequency_sweeps(self, edb_examples):
+        data = {
+            "setups": [
+                {
+                    "name": "hfss_setup_1",
+                    "type": "HFSS",
+                    "f_adapt": "5GHz",
+                    "max_num_passes": 10,
+                    "max_mag_delta_s": 0.02,
+                    "freq_sweep": [
+                        {
+                            "name": "sweep1",
+                            "type": "Interpolation",
+                            "frequencies": [
+                                {"distribution": "linear scale", "start": "50MHz", "stop": "200MHz", "step": "10MHz"}
+                            ],
+                        },
+                        {
+                            "name": "sweep2",
+                            "type": "Interpolation",
+                            "frequencies": [
+                                {"distribution": "log scale", "start": "1KHz", "stop": "100kHz", "samples": 10}
+                            ],
+                        },
+                        {
+                            "name": "sweep3",
+                            "type": "Interpolation",
+                            "frequencies": [
+                                {"distribution": "linear count", "start": "10MHz", "stop": "20MHz", "points": 11}
+                            ],
+                        },
+                    ],
+                },
+            ]
+        }
+        edbapp = edb_examples.get_si_verse()
+        assert edbapp.configuration.load(data, apply_file=True)
+        # assert len(edbapp.setups["hfss_setup_1"].frequency_sweeps["sweep1"].frequencies) == 16
+        # assert len(edbapp.setups["hfss_setup_1"].frequency_sweeps["sweep2"].frequencies) == 20
+        # assert len(edbapp.setups["hfss_setup_1"].frequency_sweeps["sweep3"].frequencies) == 11
```

Comment:
Should we remove this comments or uncomment the lines of code ?

Reply:
Yes I agree this should be removed

Reply:
done